coordinator-common/src/main/java/org/apache/kafka/coordinator/common/runtime/CoordinatorLoaderImpl.java

```diff
@@ -100,17 +100,17 @@ public CompletableFuture<LoadSummary> load(TopicPartition tp, CoordinatorPlaybac
     }
 
     private void doLoad(
-            TopicPartition tp,
-            CoordinatorPlayback<T> coordinator,
-            CompletableFuture<LoadSummary> future,
-            long startTimeMs
+        TopicPartition tp,
+        CoordinatorPlayback<T> coordinator,
+        CompletableFuture<LoadSummary> future,
+        long startTimeMs
     ) {
         long schedulerQueueTimeMs = time.milliseconds() - startTimeMs;
         try {
             Optional<UnifiedLog> logOpt = partitionLogSupplier.apply(tp);
             if (logOpt.isEmpty()) {
                 future.completeExceptionally(new NotLeaderOrFollowerException(
-                        "Could not load records from " + tp + " because the log does not exist."));
+                    "Could not load records from " + tp + " because the log does not exist."));
```

Comment:
@dajac This PR is already good to go, but there are other similar issues in this file. Do you plan to fix them all at once?

Reply:
Let me double check. I may have missed some.